aboutme.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./assets/css/reset.css">
+    <link rel="stylesheet" href="./assets/css/style.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <h1>Kevin LaCarrubba</h1>
+
+        <nav>
+            <ul>
+                <li><a href="aboutme.html">About Me</a></li>
+                <li><a href="mywork.html">My Work</a></li>
+                <li><a href="#">Resume</a></li>
+            </ul>
+        </nav>
+    </header>
+    <main>
+            <img src="./assets/images/Me.jpg" alt="Me">
+            <h2>About Me</h2>
+            <p> My name is Kevin LaCarrubba. I am a 29 years old for another 6 days. I live in Cliffside Park NJ. I recently graduated from Bergen Community College in May of 2021. I am currently attending Rutgers Coding Bootcamp, with the goal to become a front end developer. </p>
+    </main>
+    <section class="contact">
+        <a href="mailto:Kevin.LaCarrubba@yahoo.com"><img src="./assets/images/malwhite.png" alt="Email"></a>
```

Comment:
You need to close this `img` tag. Right now you have it open, with no closing at all. so do `<img src="..." alt="..." />`